pkg/install/install.go

```diff
@@ -103,55 +79,9 @@ func (i Installer) Install(d *deployment.Deployment) (err error) {
 		}
 	}
 
-	err = i.t.Init(*d)
+	err = i.u.Upgrade(d)
```

Comment:
Install is a particular case of upgrade with this change. The upgrade procedure does not require a previous snapshot to be present.